pyproject.toml

```
@@ -15,7 +15,7 @@ authors = [
 ]
 license = {text = "Apache-2.0"}
 readme = "README.md"
-requires-python = "~=3.9"
+requires-python = ">=3.12"
```

Should we change this back as well?
```suggestion
requires-python = "~=3.9"
```

---

thanks for catching this!